weldx/time.py

```diff
@@ -280,7 +342,7 @@ def __iter__(self):
         """Use generator to iterate over index values."""
         return (t for t in self.as_pandas_index())
 
-    def __getitem__(self, item):
+    def __getitem__(self, item) -> Union[Timedelta, Timestamp]:
```

Comment:
I think slicing like `[2:2:-1]` should also work and result in an `Index` type?

Reply:
Have not tested it. Is that the case?

Reply:
test it please and update accordingly

Reply:
~~~ python
from weldx import Time

t_delta = Time(["1s", "3s", "4s", "5s"])
t_abs = Time(["2000", "2001", "2002", "2003"])

print(type(t_delta[2:2:-1]))
print(type(t_abs[2:2:-1]))
~~~

output

~~~
<class 'pandas.core.indexes.timedeltas.TimedeltaIndex'>
<class 'pandas.core.indexes.datetimes.DatetimeIndex'>
~~~

good finding 👍